frontend/src/components/layout/views/CodeView.jsx

```diff
@@ -0,0 +1,67 @@
+// Standard React components
+import React from 'react'; 
+
+// DEPENDENCIES
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import SyntaxHighlighter from 'react-syntax-highlighter';
+import {androidstudio} from 'react-syntax-highlighter/dist/esm/styles/hljs';
+import 'whatwg-fetch';

```

Comment:
Why are we importing this here? We don't seem to be making any fetch requests within this file.